### ISSUE TITLE
Youtube videos sometimes autoplay

### ISSUE THREAD

**Issue by newtrishacm** [Eloston/disable-html5-autoplay#88]
The extension was working beautifully for a long while but then in the last few weeks, Youtube videos have been randomly autoplaying.  I get an 'up next' and cancel option for the autoplay but it's annoying to have to do that.  Any way around this?

**Comment by Eloston** [comment]
The next video autoplays after the up next timeout? That's weird. Thanks for reporting.

**Comment by newtrishacm** [comment]
Yeah, it happened kinda weirdly too.  At first it was only an occasional video, so I didn’t really think much of it, but now it’s pretty much every video that does it.

Take care,


Trisha Chetek-Motiuk
SAIT School of Business Graduate
Business Administration
Supply Chain Management and
General Management
[LinkedIn Button - 01]<https://ca.linkedin.com/pub/trisha-chetek-motiuk/3b/876/3b5>
